ArduPlane/tiltrotor.cpp

```diff
@@ -309,7 +309,7 @@ void Tiltrotor::continuous_update(void)
         // Q_TILT_MAX. Anything above 50% throttle gets
         // Q_TILT_MAX. Below 50% throttle we decrease linearly. This
         // relies heavily on Q_VFWD_GAIN being set appropriately.
-       float settilt = constrain_float((SRV_Channels::get_output_scaled(SRV_Channel::k_throttle)-MAX(plane.aparm.throttle_min.get(),0)) * 0.02, 0, 1);
+       float settilt = .01f * quadplane.forward_throttle_pct();
```

Comment:
We will have to do something a little more complex here to maintain the current behavior. Maybe something like:
```suggestion
       const float throttle = SRV_Channels::get_output_scaled(SRV_Channel::k_throttle);
       float settilt = 0.0f
       if (throttle > 0) {
            // Remove throttle min
            settilt = constrain_float(throttle -MAX(plane.aparm.throttle_min.get(),0)) * 0.02, 0, 1);
       } elseif (fwd_throttle_min < 0) {
           settilt = constrain_float(throttle * 0.02, 0, 1);
       }




```